nornir/plugins/inventory/netbox.py

```diff
@@ -135,6 +137,8 @@ def __init__(
         ssl_verify: Union[bool, str] = True,
         flatten_custom_fields: bool = False,
         filter_parameters: Optional[Dict[str, Any]] = None,
+        nb_private_key: Optional[str] = None,
```

Comment:
Can you clarify further here that this can be a string or a path to a file

Reply:
I clarified in the docstring, but I don't think I can get any more specific with the type hint as it is just a string when the user passes it in.

`nb_private_key: Netbox user private key file path`

Good?

Reply:
This is sill not clear to me. Do you want nb_private_key to represent a path to a file that contains the key, or could it also be a string that holds the key?

Reply:
It could be either. I clarified the documentation to show either and the logic will load it from either file or environment variable.

```
        nb_private_key: Netbox user private key. A file path if loading from disk, or a string contained within the environment
            variable NB_PRIVATE_KEY
```

Reply:
i think this is sufficiently resolved.